rock_gazebo.orogen

```diff
@@ -81,3 +81,31 @@ task_context "LaserScanTask" do
     output_port "laser_scan_samples", "/base/samples/LaserScan"
 end
 
+task_context "CameraTask" do
+    needs_configuration
+
+    subclasses 'BaseTask'
+
+    # Camera full name
+    attribute "name", "/std/string"
+
+    # Camera output port
+    output_port "frame", "/base/samples/frame/Frame"
+end
+
+task_context "ImuTask" do
+    needs_configuration
+
+    subclasses 'BaseTask'
+
+    # IMU full name
+    attribute "name", "/std/string"
+    property('imu_frame', '/std/string', 'imu').
+    	doc 'the name of the frame attached to the IMU'
+    property('world_frame', '/std/string', 'imu_world').
+    	doc 'the name of the world frame'
+
+    output_port('orientation_samples', '/base/samples/RigidBodyState').
+        doc 'provides timestamped IMUReading samples containing the orientation estimate as reported by the IMU.'
```

Comment:
Write documentation as comment above the element, e.g.

```
# Provides timestamped ...
output_port 'orientation_samples'
```

PS: the documentation string is obviously completely wrong ...